### PR TITLE
add-link-on-list-certificates

### PR DESCRIPTION
**Pull request by Goven112**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by hryhoriivoloshyn — APPROVED** [review]

File: src/pages/userPage/personalData/Course.tsx [hunk 1]
```diff
@@ -111,28 +126,45 @@ export default function () {
                     type="primary"
                     
                     className="buttonaddcertificate"
-                    onClick={() => getButton()
+                    onClick={() => showUploadWindow()
```
Comment (tvardero) [inline src/pages/userPage/personalData/Course.tsx:129]: Теж, зайва функція, можно було зробити `() => setvisibleAchievementModal(true)`

**Review by tvardero — CHANGES_REQUESTED** [review]

File: src/pages/userPage/personalData/Course.tsx [hunk 2]
```diff
@@ -15,44 +15,61 @@ import { getAllCourse, getAllCourseByUserId } from "../../../api/courseApi";
...
   const [visibleAchievementModal, setvisibleAchievementModal] = useState(false);
-
+  const [visibleListModal, setvisibleListModal] = useState(false);
   const [showAchievementModal, setshowAchievementModal] = useState(false);
   const [isDataLoaded, setDataLoaded] = useState<boolean>(false);
-
+  
   const [allCourses, setallCourses] = useState<Course[]>([]);
+  const [achievementDoc, setAchievementDoc] = useState<BlankDocument[]>([]);
+  const [userToken, setUserToken] = useState<any>([
```
[the remark this answers is not in the document]
Comment (Goven112) [reply src/pages/userPage/personalData/Course.tsx:32]: ну це в іншому функціоналі так само зроблено тому я так само зробив

**Review by Serhii-Buniak — APPROVED** [review]

File: src/pages/userPage/personalData/Course.tsx [hunk 3]
```diff
@@ -111,28 +126,45 @@ export default function () {
                     type="primary"
                     
                     className="buttonaddcertificate"
-                    onClick={() => getButton()
+                    onClick={() => showUploadWindow()
```
[the remark this answers is not in the document]
Comment (Goven112) [reply src/pages/userPage/personalData/Course.tsx:130]: deleted paragraphs

File: src/pages/userPage/personalData/Course.tsx (hunk 1 above)
Comment (Goven112) [reply src/pages/userPage/personalData/Course.tsx:129]: done

File: src/pages/userPage/personalData/Course.tsx [hunk 4]
```diff
@@ -15,44 +15,61 @@ import { getAllCourse, getAllCourseByUserId } from "../../../api/courseApi";
...
 
 
   const urlaim = `/userpage/blank/${activeUserId}`;
-  const getButton = async () => {
+
+  const showUploadWindow = async () => {
     setvisibleAchievementModal(true);
-   
+  };
+  const showListCertificates = async () => {
```
[the remark this answers is not in the document]
Comment (Goven112) [reply src/pages/userPage/personalData/Course.tsx:57]: done

File: src/pages/userPage/personalData/Course.tsx [hunk 5]
```diff
@@ -15,42 +15,53 @@ import { getAllCourse, getAllCourseByUserId } from "../../../api/courseApi";
...
 export default function () {
 
   const [visibleAchievementModal, setvisibleAchievementModal] = useState(false);
-
+  const [visibleListModal, setvisibleListModal] = useState(false);
   const [showAchievementModal, setshowAchievementModal] = useState(false);
   const [isDataLoaded, setDataLoaded] = useState<boolean>(false);
 
   const [allCourses, setallCourses] = useState<Course[]>([]);
+  const [achievementDoc, setAchievementDoc] = useState<BlankDocument[]>([]);
```
Comment (tvardero) [inline src/pages/userPage/personalData/Course.tsx:31]: Заміни на звичайну змінну

**Review by tvardero — CHANGES_REQUESTED** [review]

**Review by tvardero — APPROVED** [review]

**Review by tvardero — CHANGES_REQUESTED** [review]
Wait

**Review by tvardero — APPROVED** [review]

**Pull request merged**